packages/web/docs/src/components/icons.tsx

```diff
@@ -0,0 +1,17 @@
+// these are different than CheckIcon and CloseIcon we have in the design system
+
+export function CheckmarkIcon(props: React.SVGProps<SVGSVGElement>) {
+  return (
+    <svg width="16" height="16" viewBox="0 0 16 16" fill="currentColor" {...props}>
+      <path d="M6.66668 10.1134L12.7947 3.98608L13.7373 4.92875L6.66668 11.9994L2.42401 7.75675L3.36668 6.81408L6.66668 10.1134Z" />
+    </svg>
+  );
+}
+
+export function XIcon(props: React.SVGProps<SVGSVGElement>) {
```

Comment:
"X" for "no support" and "X" for "close" look different and that kinda makes sense.